services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: list[str] = None,
```

Comment:
The `title` parameter should be annotated as `str` or `Optional[str]` instead of `list[str]`, as it is used as a single string in the filter.